components/actionmenu/stories/actionmenu.stories.js

```diff
@@ -89,3 +89,11 @@ export default {
 
 export const Default = Template.bind({});
 Default.args = {};
+Default.parameters = {
+	docs: {
+		story: {
+			inline: false,
+			iframeHeight: 250,
```

Comment:
Oh interesting!  Great job catching this fix. Is this a best practice we should document for any component using a popover or that uses a certain type of positioning?

Reply:
This is the first time we've used it, but I do think it will be necessary to use on the Docs page of any component that uses absolute positioning. It helps us ensure that the whole component can show in the preview frame for Docs.